spec/schemas/_common.yaml

```diff
@@ -672,25 +672,25 @@ components:
         vector:
           $ref: '#/components/schemas/QueryVector'
         k:
-          description: The final number of nearest neighbors to return as top hits.
+          description: The final number of nearest neighbors to return as top hits. 
           type: number
         min_score:
-          description: The minimum similarity score for a neighbor to be considered a hit.
+          description: The minimum similarity score for a neighbor to be considered a hit. 
           type: number
           x-version-added: '2.14'
         max_distance:
-          description: The maximum physical distance in vector space for a neighbor to be considered a hit.
+          description: The maximum physical distance in vector space for a neighbor to be considered a hit. 
```

Comment:
"The maximum physical vector space distance required in order for a neighbor to be considered a hit"?

Reply:
That works